### ISSUE TITLE
Form submission results in 500 error

### ISSUE THREAD

**Issue by andrewzah** [kemalyst/kemalyst#13]
```
ERROR: Missing hash key: "username" (KeyError)
```
```
0x5f60d4: *Hash(String, Array(String)) at /opt/crystal/src/hash.cr 123:9
0x5f5fbd: *Hash(String, Array(String)) at /opt/crystal/src/hash.cr 60:5
0x5f6935: *HTTP::Params#[]<String>:String at /opt/crystal/src/http/params.cr 117:7
0x5c5f27: *SessionController::Create#call<HTTP::Server::Context>:String at /mnt/c/Users/andrew/Desktop/Sync/_programming/Crystal/main_site/src/controllers/session_controller.cr 15:7
0x5c3777: *Kemalyst::Handler::Router#process_request<HTTP::Server::Context>:HTTP::Server::Context at /mnt/c/Users/andrew/Desktop/Sync/_programming/Crystal/main_site/lib/kemalyst/src/kemalyst/handler/router.cr 157:26
0x5c2dc5: *Kemalyst::Handler::Router#call<HTTP::Server::Context>:HTTP::Server::Context at /mnt/c/Users/andrew/Desktop/Sync/_programming/Crystal/main_site/lib/kemalyst/src/kemalyst/handler/router.cr 114:7
0x5d3738: *Kemalyst::Handler::Params at /opt/crystal/src/http/server/handler.cr 22:7
0x5d24d0: *Kemalyst::Handler::Params#call<HTTP::Server::Context>:(Array(IO) | Bool | HTTP::Server::Context | HTTP::Server::Response | HTTP::Server::Response::Output | IO::FileDescriptor+ | Int32 | String | Nil) at /mnt/c/Users/andrew/Desktop/Sync/_programming/Crystal/main_site/lib/kemalyst/src/kemalyst/handler/params.cr 22:7
0x5d1a66: *Kemalyst::Handler::Flash at /opt/crystal/src/http/server/handler.cr 22:7
0x5d118f: *Kemalyst::Handler::Flash#call<HTTP::Server::Context>:HTTP::Server::Context at /mnt/c/Users/andrew/Desktop/Sync/_programming/Crystal/main_site/lib/kemalyst/src/kemalyst/handler/flash.cr 25:22
0x5ccde1: *Kemalyst::Handler::Session at /opt/crystal/src/http/server/handler.cr 22:7
0x5cc39f: *Kemalyst::Handler::Session#call<HTTP::Server::Context>:HTTP::Server::Context at /mnt/c/Users/andrew/Desktop/Sync/_programming/Crystal/main_site/lib/kemalyst/src/kemalyst/handler/session.cr 27:22
0x5ce127: *Kemalyst::Handler::Static at /opt/crystal/src/http/server/handler.cr 22:7
0x5cd8a1: *Kemalyst::Handler::Static#call<HTTP::Server::Context>:(Array(IO) | Bool | HTTP::Server::Context | HTTP::Server::Response | HTTP::Server::Response::Output | IO::FileDescriptor+ | Int32 | String | Nil) at /mnt/c/Users/andrew/Desktop/Sync/_programming/Crystal/main_site/lib/kemalyst/src/kemalyst/handler/static.cr 22:9
0x5d04e0: *Kemalyst::Handler::Error at /opt/crystal/src/http/server/handler.cr 22:7
0x5cfe2c: *Kemalyst::Handler::Error#call<HTTP::Server::Context>:(Array(IO) | Bool | HTTP::Server::Context | HTTP::Server::Response | HTTP::Server::Response::Output | IO::FileDescriptor+ | Int32 | String | Nil) at /mnt/c/Users/andrew/Desktop/Sync/_programming/Crystal/main_site/lib/kemalyst/src/kemalyst/handler/error.cr 14:9
0x5cf30f: *Kemalyst::Handler::Logger at /opt/crystal/src/http/server/handler.cr 22:7
0x5cede2: *Kemalyst::Handler::Logger#call<HTTP::Server::Context>:HTTP::Server::Context at /mnt/c/Users/andrew/Desktop/Sync/_programming/Crystal/main_site/lib/kemalyst/src/kemalyst/handler/logger.cr 15:21
0x5d4a61: *HTTP::Server::RequestProcessor#process<(OpenSSL::SSL::Socket::Server | TCPSocket+), (OpenSSL::SSL::Socket::Server | TCPSocket+), IO::FileDescriptor>:Nil at /opt/crystal/src/http/server/request_processor.cr 39:11
0x5d45f9: *HTTP::Server::RequestProcessor#process<(OpenSSL::SSL::Socket::Server | TCPSocket+), (OpenSSL::SSL::Socket::Server | TCPSocket+)>:Nil at /opt/crystal/src/http/server/request_processor.cr 16:3
0x5d44c9: *HTTP::Server#handle_client<(TCPSocket+ | Nil)>:Nil at /opt/crystal/src/http/server.cr 174:5
0x4d3ee3: ~procProc(Nil) at /opt/crystal/src/concurrent.cr 60:3
0x4eb7b4: *Fiber#run:(IO::FileDescriptor | Nil) at /opt/crystal/src/fiber.cr 114:3
0x4cb8f6: ~proc2Proc(Fiber, (IO::FileDescriptor | Nil)) at /opt/crystal/src/concurrent.cr 60:3
0x0: ??? at ??
```

This was previously working so I imagine it has to do with a recent update.

**Comment by drujensen** [comment]
Thanks for reporting this.  I will take a look soon.

**Comment by drujensen** [comment]
Released 0.2.7
